inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -243,7 +243,12 @@ class V10Parser : public IParser {
             } else if (auto a = ngraph::as_type<ngraph::AttributeAdapter<ngraph::op::TopKMode>>(&adapter)) {
                 if (!getStrAttribute(node.child("data"), name, val)) return;
                 static_cast<ngraph::op::TopKMode&>(*a) = ngraph::as_enum<ngraph::op::TopKMode>(val);
-            }  else {
+            } else if (auto a = ngraph::as_type<ngraph::AttributeAdapter<ngraph::CoordinateDiff>>(&adapter)) {
```

Comment:
Why do we need an adapter for `CoordinateDiff` in this PR?

Reply:
It is required by visit_attributes for Convolution op, as the type of "m_pads_begin" and "m_pads_end" is CoordinateDiff.

Reply:
Why this PR contains changes which are required for Convolution op?

Reply:
Right, coordinateDiff is part of Convolution op, not needed for Const implementation. Removed from this PR.